lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -159,12 +153,63 @@ void AmfNasStateManager::amf_nas_state_init_local_state() {
  */
 amf_app_desc_t* AmfNasStateManager::get_state(bool read_from_redis) {
   state_dirty = true;
-
+  if (persist_state_enabled && read_from_redis) {
+    read_state_from_db();
+  }
   return state_cache_p;
 }
 
 map_uint64_ue_context_t AmfNasStateManager::get_ue_state_map() {
   return state_ue_map;
 }
 
+status_code_e AmfNasStateManager::read_state_from_db() {
+#if !MME_UNIT_TEST
+  StateManager::read_state_from_db();
+#else
+  if (persist_state_enabled) {
+    magma::lte::oai::MmeNasState state_proto = magma::lte::oai::MmeNasState();
```

Comment:
add `using magma::lte...` directive at top of the file and update this and other usages accordingly

Reply:
Review comment addressed.